mr/src/test/java/org/apache/iceberg/mr/mapred/serde/objectinspector/TestIcebergObjectInspector.java

```diff
@@ -38,31 +38,37 @@
 
 public class TestIcebergObjectInspector {
 
+  private int id = 0;
```

Comment:
Why was this introduced? It seems like relying on the same execution order between the schema creation and the test methods is brittle.

I'd prefer to move back to fixed IDs since that's easier to test and more clear in assertions.

Reply:
That made my life easier when adding new field but I get your point, I'll fix it in a follow-up PR.

Reply:
@rdblue , @guilload, @massdosage if there are many issues, open items does it make sense to create a milestone with all the open tickets? So that it can be worked on in parallel by us and we don't duplicate effort or step on each other's toes?

Reply:
Sounds good to me. You should be able to create and edit milestones.

Reply:
I think the only missing necessary follow up here already has a PR at #1157 so I'm not sure we need to do this? We're now co-ordinating with @guilload on the next steps for the `mapred` InputFormat.